worker/src/main/java/gov/cms/ab2d/worker/processor/PatientClaimsProcessorImpl.java

```diff
@@ -39,27 +48,109 @@
     @Value("${bfd.earliest.data.date:01/01/2020}")
     private String earliestDataDate;
 
+    @Value("${aggregator.directory.finished}")
```

Comment:
Same thought that there should be a class for these variables to simplify things.

Reply:
Will do